ranger-http/src/main/java/io/appform/ranger/http/serviceprovider/HttpNodeDataSink.java

```diff
@@ -40,10 +40,12 @@
 public class HttpNodeDataSink<T, S extends HttpRequestDataSerializer<T>> extends HttpNodeDataStoreConnector<T> implements NodeDataSink<T, S> {
 
     private final Service service;
+    private final ObjectMapper mapper;
 
-    public HttpNodeDataSink(Service service, HttpClientConfig config, ObjectMapper mapper, OkHttpClient httpClient) {
-        super(config, mapper, httpClient);
+    public HttpNodeDataSink(Service service, HttpClientConfig config, ObjectMapper mapper, HttpCommunicator<T> httpClient) {
```

Comment:
We can get away with explicit httpClientConfig, when we have the communicator no? The communicator inherently should have an httpClientConfig.

Reply:
Applicable in multiple places, perhaps?

Reply:
this was already there and it goes to the base. Removed mapper from base and added it here.